test.js

```diff
@@ -24,4 +24,7 @@ function foo()  {}
 
 function(){}
 function foo (){}
-function foo ()  {}
\ No newline at end of file
+function foo ()  {}
+
+console.log('hello, world!');
+
```

Comment:
one could remove the newline, but the console log tests the eol change this pr introduces